discovery-provider/src/tasks/entity_manager/entity_manager.py

```diff
@@ -216,6 +218,12 @@ def entity_manager_update(
                         and ENABLE_DEVELOPMENT_FEATURES
                     ):
                         view_playlist(params)
+                    elif (
+                        params.action == Action.CREATE
+                        and params.entity_type == EntityType.DELEGATE
+                        and ENABLE_DEVELOPMENT_FEATURES
```

Comment:
Is `ENABLE_DEVELOPMENT_FEATURES` still needed?

Reply:
nah we should clean this up